schema.sql

```diff
@@ -1,10 +1,11 @@
 /* CREATING THE ANIMALS TABLE */
 
 CREATE TABLE animals (
-   id int,
-   name VARCHAR(50),
-   date_of_birth DATE,
+   id int PRIMARY KEY,
+   name VARCHAR(50) NOT NULL,
+   date_of_birth DATE NOT NULL,
     escape_attempts INT,
     neutered BOOLEAN,
-    weigth_kg DECIMAL
+    weigth_kg DECIMAL,
+    species VARCHAR(50)
```

Comment:
### Optional
- [ ] It would be better if we use the `ALTER` keyword to make changes to our schema instead of adding the columns as we have it. For example; `ALTER TABLE animals ADD species varchar(255);`

Reply:
NOTED WITH THANKS